### ISSUE TITLE
Make fails

### ISSUE THREAD

**Issue by KonradHoeffner** [openlink/virtuoso-opensource#255]
After running `./autogen.sh` and `./configure`, make fails with the following output (I also made the adjustments given for Linux 64). My machine is a `3.17.2-1-ARCH #1 SMP PREEMPT Thu Oct 30 20:49:39 CET 2014 x86_64 GNU/Linux` and configure doesn't report problems.

```
getdate.c: In function 'OPL_gd_parse':
../../libsrc/util/getdate.y:74:17: error: too few arguments to function 'OPL_gd_lex'
 #define yylex   OPL_gd_lex
                 ^
getdate.c:1368:16: note: in expansion of macro 'yylex'
       yychar = yylex (&yylval);
                ^
../../libsrc/util/getdate.y:74:17: note: declared here
 #define yylex   OPL_gd_lex
                 ^
../../libsrc/util/getdate.y:113:12: note: in expansion of macro 'yylex'
 static int yylex (void *yylval, void *cookie);
            ^
../../libsrc/util/getdate.y:169:30: error: 'cookie' undeclared (first use in this function)
 #define context ((CONTEXT *) cookie)
                              ^
../../libsrc/util/getdate.y:201:6: note: in expansion of macro 'context'
      context->yyHaveTime++;
      ^
../../libsrc/util/getdate.y:169:30: note: each undeclared identifier is reported only once for each function it appears in
 #define context ((CONTEXT *) cookie)
                              ^
../../libsrc/util/getdate.y:201:6: note: in expansion of macro 'context'
      context->yyHaveTime++;
      ^
../../libsrc/util/getdate.y: In function 'get_date':
../../libsrc/util/getdate.y:73:17: error: too many arguments to function 'OPL_gd_parse'
 #define yyparse OPL_gd_parse
                 ^
../../libsrc/util/getdate.y:938:7: note: in expansion of macro 'yyparse'
   if (yyparse (&cookie)
       ^
../../libsrc/util/getdate.y:73:17: note: declared here
 #define yyparse OPL_gd_parse
                 ^
getdate.c:1199:1: note: in expansion of macro 'yyparse'
 yyparse (void)
 ^
```

**Comment by HughWilliams** [comment]
What is the Linux distribution being used (`uname -a` & `cat /etc/issue`)? This looks like a known issue with latest Ubuntu 14.x version, as detailed in #160.

You should try building with the latest `develop/7` where a fix for this is available...

**Comment by KonradHoeffner** [comment]
I use Arch Linux instead of Ubuntu but using `develop/7` also allowed me to successfully compile it, many thanks!

**Comment by johentsch** [comment]
I'm trying to install VOS (`stable/6`) on a Raspberry Pi (for reasons) and I've run into the same issue. Unfortunately, I cannot use `stable/7` instead because it doesn't work on 32bit systems such as

    Linux raspberrypi 4.19.75-v7+ #1270 SMP Tue Sep 24 18:45:11 BST 2019 armv7l GNU/Linux
    Raspbian GNU/Linux 10 \n \l

bison == 3.3.2

Would you need any further information?

**Comment by TallTed** [comment]
@laserjeyes - I suggest you create a new issue for your situation, with reference to thiS (#255).
